src/ts/src/common/types/bioinformatics.d.ts

```diff
@@ -4,6 +4,7 @@
 
 interface Sample {
```

Comment:
should there be a base interface for anything that can show up in a table view and have the type field live there?